### ISSUE TITLE
Countries button accessibility props not translated

### ISSUE THREAD

**Issue by hstandaert** [AstrOOnauta/react-native-international-phone-number#142]
When using screenreaders like Talkback and/or Voiceover, the accessibility label and hint for the button that triggers the modal cannot be translated. It is always read in English.

**Desired solution**

- A prop that allows us to override the message that is read when focusing the modal trigger
- Have it automatically translated when the `language` prop is passed in

**Comment by AstrOOnauta** [comment]
Hey @hstandaert , thanks for your issue.

I will improve this and release a new version soon.

**Comment by AstrOOnauta** [comment]
Hello @hstandaert,

I release a new version fixing this issue, agree your desired solution. So, please, update your lib for the [latest version v.0.10.7](https://www.npmjs.com/package/react-native-international-phone-number/v/0.10.7)

Let me know if it does not works well...

🚀 AstrOOnauta 🚀
